src/model/queries/ticket.js

```diff
@@ -48,16 +48,16 @@ const getNotSolvedCount = () => {
 
 const getSupports = () => {
```

Comment:
This query gets all the ```unsolved``` tickets.
What would you name the query to get ```solved``` tickets?

I think it's important that the name of the function reflects what it does.

Reply:
This query gets all ticket that have a support type and all status except solved one